frontends/web/src/routes/settings/manage-accounts.tsx

```diff
@@ -226,6 +253,25 @@ class ManageAccounts extends Component<Props, State> {
                                 </div>
                             </div>
                         ) : null }
+                        { editAccountCode ? (
+                            <Dialog
+                                onClose={() => this.setState({ editAccountCode: undefined, editAccountNewName: '' })}
+                                title={t('manageAccounts.editAccountNameTitle')}>
+                                <form onSubmit={this.updateAccountName}>
+                                    <Message type="warning" hidden={!editErrorMessage}>
+                                        {editErrorMessage}
+                                    </Message>
+                                    <Input
+                                        onInput={e => this.setState({ editAccountNewName: e.target.value})}
```

Comment:
space before `}`? :D

Reply:
Looking at it I don't like it, but commit it anyway. 
![Screen Shot 2021-05-20 at 4 25 01 PM](https://user-images.githubusercontent.com/546900/118996178-fe1d7900-b987-11eb-9d78-1843ffa43070.png)

Reply:
imho whose two {} are different, one is TS and the other is TSX. we don't usually put spaces in JSX {}, but we do for JS objects. anyway :D

Reply:
there is like 50 `}` no wonder I missed it :P